tarentula_documents.csv

```diff
@@ -0,0 +1 @@
+query,documentUrl,documentId,rootId,documentNumber,metadata.tika_metadata_creation_date,metadata.tika_metadata_author
```

Comment:
I don't see where this CSV is used in the PR. Is it necessary to add it?

Reply:
ups, it should not be there actually, I remove it